### PR TITLE
primehack.primehack version 01

### PR DESCRIPTION
**Pull request by MonocleMAYHEM18**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/119245)

### CODE REVIEW

**Comment by MonocleMAYHEM18** [comment]
yes

**Comment by MonocleMAYHEM18** [comment]
@microsoft-github-policy-service agree [company="{decamillennium}"]
@microsoft-github-policy-service agree
@microsoft-github-policy-service agree company="Microsoft"

**Comment by stephengillie** [comment]
> @microsoft-github-policy-service agree [company="{decamillennium}"] @microsoft-github-policy-service agree @microsoft-github-policy-service agree company="Microsoft"

I think you want the middle one. Please only put one so the robot doesn't get confused.

File: manifests/p/primehack/primehack/01/primehack.primehack.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: primehack.primehack
+PackageVersion: "01"
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: Dolphin.exe
+  PortableCommandAlias: primal
```
Comment (stephengillie) [inline manifests/p/primehack/primehack/01/primehack.primehack.installer.yaml:10]:
```suggestion
  PortableCommandAlias: primal
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

Comment (stephengillie) [reply manifests/p/primehack/primehack/01/primehack.primehack.installer.yaml:10]: This package installs normally, but gives this message when executed:
image

**Pull request closed without merging**